hive-connector/src/main/java/org/apache/iotdb/hive/TSFHiveInputFormat.java

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
We are using block comments style for Apache License instead of Java-style comments during to [this PR](https://github.com/apache/incubator-iotdb/pull/420). Maybe you could change all the new files' Apache License in next PR, or we may face some problems when generating JavaDoc.